ansible/roles/monitoring-influxdb/vars/main.yml

```diff
@@ -5,6 +5,8 @@ influx_data_dir: "{{ base_monitoring_dir }}/influxdb"
 
 influxdb_version: "0.10.1-1"
 
+downsample_data: false
```

Comment:
Rename as above.
Can you also add a comment like `Needed for first install. Subsequently setting this to true will drop existing data (if any.)`